This fixes begriffs/angular-paginate-anything#68

tafs7 opened the pull request:
since directive is opinionated on the Range-Unit being "items", we'll
make sure the content header includes the unit value.
Also updated the README file to address this.

The simple little change to the Regex that matches the header value now ensures this works according to the RFC spec.

This is a breaking change from the previous behavior for anyone who used this in the past as the README instructed.

Ideally, we could expose an optional property from the directive to allow folks to set the unit used by the Range-Unit header, which defaults to "items".  Then use that value in the regex, instead of a hardcoded value.

begriffs commented:
Thanks for making this standards compliant. I overlooked that part of the spec.

Let's make the directive accept both `items x-y/z` and `x-y/z` but not `bytes x-y/z` or other units. (The old "be conservative in what you send, be liberal in what you accept.")

tafs7 commented:
@begriffs I've updated the regex to allow both `items x-y/z` and `x-y/z` as requested.  I've also updated the README to include an updated link for the ServiceStack sample, and a link to the new Web API sample, which @dnauck requested I provide in the wiki.

dnauck commented:
@tafs7 thank you very much
